auditbeat/docs/fields.asciidoc

```diff
@@ -6070,6 +6070,28 @@ These are the fields generated by the system module.
 
 
 
+
+*`event.origin`*::
++
+--
+type: keyword
+
+Origin of the event. This can be a file path (e.g. `/var/log/log.1`), or the name of the system component that supplied the data (e.g. `netlink`).
+
+
+--
+
+
+*`user.terminal`*::
```

Comment:
I think it would be good to open a PR to elastic/ecs suggesting that this be added. I think the auditd module gives tty info.

Reply:
It does. https://github.com/elastic/beats/pull/10192 will fill `process.terminal` for the Filebeat auditd module. I wonder if we should use the same field, and whether it should be `user.terminal` or `process.terminal`. @andrewkroh @webmat what do you think?

Reply:
Yeah in my PR I've been mapping both `auditd.log.tty` and `auditd.log.terminal` to `process.terminal`, thinking it was inconsistency on the auditd side. This makes me wonder if one is actually user-specific and one process-specific. I'll dig into this some more for my PR.

But I do think `user.terminal` makes sense, if you have terminal info for users.

Reply:
Found some light reading for the week-end: [Auditd field dictionary](https://github.com/linux-audit/audit-documentation/blob/master/specs/fields/field-dictionary.csv)